install/update.php

```diff
@@ -836,7 +837,7 @@ function updateDbUpTo031() {
    }
 
    // Update version number and default langage and new version_founded ---- LEAVE AT THE END
-   Config::setConfigurationValues('core', array('version'             => GLPI_VERSION,
+   Config::setConfigurationValues('core', array('version'             => GLPI_SCHEMA_VERSION,
```

Comment:
I'm not sure about this one.
With an example, if glpi_version = 9.2.2 and schema version is 9.2, the update will always be triggered, no ?